### PR TITLE
setup localdatabase

### PR DESCRIPTION
**Pull request by gconnect**
## What this PR does
- Setup the local database using Room persistent library

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -112,4 +112,18 @@ dependencies {
...
+    // Lifecycle components
+    implementation "androidx.lifecycle:lifecycle-extensions:$version_lifecycle"
+    kapt "androidx.lifecycle:lifecycle-compiler:$version_lifecycle"
+    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:$version_lifecycle"
+    implementation "androidx.lifecycle:lifecycle-livedata-ktx:$version_lifecycle"
+    implementation "androidx.lifecycle:lifecycle-runtime-ktx:$version_lifecycle"
+
+
+
+
```
Comment (ujjwalagrawal17) [inline app/build.gradle]: REMOVE EXTRA WHITELINES.

Comment (gconnect) [reply app/build.gradle]: Done

**Comment by gconnect** [comment]
I have removed the white spaces

**Comment by victorvicari** [comment]
@gconnect good job! If you are ok with this, this project uses `Koin` for `dependency injection`, can you create the `database` with `koin module`? If so, there's tutorial here, https://android.jlelse.eu/painless-android-testing-with-room-koin-bb949eefcbee
the test, aren't necessary.

**Comment by gconnect** [comment]
Okay will check it out! Thanks

**Comment by ujjwalagrawal17** [comment]
@gconnect any update on this PR ?

**Comment by gconnect** [comment]
I am still working on it. Just want to finish up with the other issues you pointed out before I go back to this

**Comment by ujjwalagrawal17** [comment]
Closing this as #287 already has this functionality.

**Pull request closed without merging**